src/zcash/Zcash.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _ZCCONSTANTS_H_
+#define _ZCCONSTANTS_H_
+
+#define ZC_NUM_JS_INPUTS 2
+#define ZC_NUM_JS_OUTPUTS 2
+#define INCREMENTAL_MERKLE_TREE_DEPTH 20
+#define INCREMENTAL_MERKLE_TREE_DEPTH_TESTING 4
+
+// TODO: these constants should be 'ZC'
+// for consistency, but I didn't want to
+// interfere with the old constants
```

Comment:
Later in this PR the other constants are removed; I will add another commit that fixes these names.